### PR TITLE
Adds more Job slots to sage

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

Opens up more Job slots

## Why It's Good For The Game

Because of the recent spike in pop on sage, people often dont spawn in the departments they wanted to and end up spawning as roles they set as never, for late joiners this has the effect that all the job slots are taken up and theres dozens of assistants, i didnt want to change this number too drastically because departments have only so much equipment, more job slots would require a map change to add more lockers for equipment.

## Changelog
:cl:
config: added more job slots to each department
config: increased ruin budgets
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
if you dont agree with these numbers i would be very happy to tweak them

File: config/Sage/jobs.txt [hunk 1]
```diff
@@ -12,12 +12,12 @@ Chief Medical Officer=1,1
 Assistant=-1,-1
 
 Quartermaster=1,1
-Cargo Technician=4,2
-Shaft Miner=3,3
+Cargo Technician=6,2
```
Comment (ike709) [inline config/Sage/jobs.txt]: I don't think you understand how this config works. The first value is the maximum including latejoins, the second is the maximum roundstart positions.

```suggestion
Cargo Technician=6,4
```

File: config/Sage/jobs.txt [hunk 2]
```diff
@@ -12,12 +12,12 @@ Chief Medical Officer=1,1
...
+Shaft Miner=4,3
```
Comment (ike709) [inline config/Sage/jobs.txt]: Make sure Mining has enough equipment for everyone if you increasing the starting value.

File: config/Sage/jobs.txt [hunk 3]
```diff
@@ -12,12 +12,12 @@ Chief Medical Officer=1,1
...
 
 Bartender=2,1
 Cook=2,1
-Botanist=3,2
+Botanist=4,2
```
Comment (ike709) [inline config/Sage/jobs.txt]:
```suggestion
Botanist=4,3
```

File: config/Sage/jobs.txt [hunk 4]
```diff
@@ -30,17 +30,17 @@ Chaplain=1,1
 Station Engineer=6,5
 Atmospheric Technician=4,2
 
-Medical Doctor=7,4
+Medical Doctor=8,4
```
Comment (ike709) [inline config/Sage/jobs.txt]:
```suggestion
Medical Doctor=8,6
```

File: config/Sage/jobs.txt [hunk 5]
```diff
@@ -30,17 +30,17 @@ Chaplain=1,1
...
 Chemist=3,2
 Geneticist=3,2
 Virologist=1,1
 
-Scientist=6,4
+Scientist=8,4
```
Comment (ike709) [inline config/Sage/jobs.txt]:
```suggestion
Scientist=8,6
```

File: config/Sage/jobs.txt [hunk 6]
```diff
@@ -30,17 +30,17 @@ Chaplain=1,1
...
 Roboticist=2,2
 
 Warden=1,1
 Detective=1,1
-Security Officer=5,5
+Security Officer=7,5
```
Comment (ike709) [inline config/Sage/jobs.txt]: Delta doesn't have the equipment for this, AFAIK.

Comment (ghost) [reply config/Sage/jobs.txt]: im gonna make it 6 then

File: config/Sage/jobs.txt (hunk 2 above)
Comment (ghost) [reply config/Sage/jobs.txt]: the mining station has an additional 3 lockers, you could have a total of 6 job slots, although crossed suggested that 4 is enough and 5 would be pushing it

File: config/Sage/jobs.txt (hunk 6 above)
Comment (ike709) [reply config/Sage/jobs.txt]: Check for the locker landmarks though to see if I'm mistaken.

File: config/Sage/jobs.txt [hunk 7]
```diff
@@ -27,20 +27,20 @@ Lawyer=2,2
...
 Virologist=1,1
 
-Scientist=6,4
+Scientist=8,6
 Roboticist=2,2
 
 Warden=1,1
 Detective=1,1
-Security Officer=5,5
+Security Officer=6,6
```
Comment (ike709) [inline config/Sage/jobs.txt]:
```suggestion
Security Officer=10,8
```
I was wrong, there's more locker landmarks in another room.

**Comment by Crossedfall** [comment]
To compensate for the increased number of miners, I'd increase these two options in the `game_options.txt` file to 75 and 20 respectively. That should provide a couple of different/larger ruins to compensate for the increased number of crew running around. 

https://github.com/BeeStation/BeeStation-Hornet/blob/af64293ba060a1cf4fb597d55850f98b6f142ead/config/Sage/game_options.txt#L525-L528

This might be reverted later on since we're already pretty close to our memory cap, but I'd be interested in seeing if we can handle the increase.

**Comment by ghost** [comment]
> 
> 
> To compensate for the increased number of miners, I'd increase these two options in the `game_options.txt` file to 75 and 20 respectively. That should provide a couple of different/larger ruins to compensate for the increased number of crew running around.
> 
> https://github.com/BeeStation/BeeStation-Hornet/blob/af64293ba060a1cf4fb597d55850f98b6f142ead/config/Sage/game_options.txt#L525-L528
> 
> This might be reverted later on since we're already pretty close to our memory cap, but I'd be interested in seeing if we can handle the increase.

done

**Comment by ghost** [comment]
any other tweaks or is this final?

**Review by Crossedfall — APPROVED** [review]
Good with me. I'll mirror and merge

**Pull request merged**